_posts/2022-05-09-Buildah-version-v1.26.1-mini.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Buildah version 1.26.1 Released!
+layout: default
+author: tsweeney
+categories: [releases]
+
+tags: community, open source, buildah, hpc, opensource, containers, images, image
+---
+![buildah logo](https://buildah.io/images/buildah.png)
+
+# Buildah version 1.26.1 Released!
+
+We're pleased to announce the release of [Buildah](https://github.com/containers/buildah) [version 1.26.1](https://github.com/containers/buildah/releases/tag/v1.26.1) and [version 1.26.0](https://github.com/containers/buildah/releases/tag/v1.26.0), which are now available from GitHub for any Linux distro.  We are shipping this release on Fedora 35, Fedora 226, and RHEL 8.6.0.2.  Buildah will also be shipped on CentOS, OpenSUSE, and Ubuntu soon.  In addition, container images will be available at https://quay.io/repository/buildah/stable and https://quay.io/repository/containers/buildah.
```

Comment:
Typo fedora 226 should be 36.

Also I believe we do not ship this in fedora 35 due the netavark changes. Fedora 35 currently has v1.23 and I don't think we should change this.